src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -51,12 +53,13 @@
     private final String folder;
     private final String customizationSpec;
     private final boolean powerOn;
+    private final int timeoutInSeconds;
     private String IP;
 
     @DataBoundConstructor
     public Clone(String sourceName, String clone, boolean linkedClone,
                  String resourcePool, String cluster, String datastore, String folder,
-                 boolean powerOn, String customizationSpec) throws VSphereException {
+                 boolean powerOn, int timeoutInSeconds, String customizationSpec) throws VSphereException {
```

Comment:
"int" should be "Integer" in order to cope with old configuration data where this parameter did not exist.